equinix/data_source_network_device.go

```diff
@@ -295,6 +295,11 @@ func createDataSourceNetworkDeviceSchema() map[string]*schema.Schema {
 			Computed:    true,
 			Description: neDeviceDescriptions["DiverseFromDeviceUUID"],
 		},
+		neDeviceSchemaNames["Tier"]: {
```

Comment:
The data source gained a `tier` attribute as well, but I don't see a docs update for that.

Reply:
added in data source(terraform-provider-equinix/docs/data-sources/network_device.md)